### PR TITLE
APIS-6541 - Integrate more complete app domain lib

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Request                                                    | Description
 `/applications?status=CREATED`                             | All Applications in testing
 `/applications?search=foo`                                 | All Applications containing the text `foo` in either their name or id
 `/applications?apiSubscription=test-service&apiVersion=v1` | All Applications subscribing to `v1` of `test-service` API
-`/applications?accessType=STANDARD&page=1&pageSize=25`     | First 25 Applications using `Standard` access type
+`/applications?accessType=STANDARD&page=1&pageSize=25`     | First 25 Applications using `Access.Standard` access type
```
Comment (peteslater-ee) [inline README.md:42]: Was this doc meant to be changed or was it part of a search n replace?

File: app/uk/gov/hmrc/thirdpartyapplication/models/ApplicationSearch.scala [hunk 2]
```diff
@@ -145,10 +145,10 @@ case object AccessTypeFilter extends AccessTypeFilter {
 
   def apply(value: String): Option[AccessTypeFilter] = {
     value match {
-      case "STANDARD"   => Some(StandardAccess)
-      case "ROPC"       => Some(ROPCAccess)
-      case "PRIVILEGED" => Some(PrivilegedAccess)
-      case _            => None
+      case "AccessType.AccessType.STANDARD" => Some(StandardAccess)
```
Comment (peteslater-ee) [inline app/uk/gov/hmrc/thirdpartyapplication/models/ApplicationSearch.scala:148]: This looks wrong. Is it wrong?

File: README.md (hunk 1 above)
Comment (AndySpaven) [reply README.md:42]: Sorted

File: app/uk/gov/hmrc/thirdpartyapplication/models/ApplicationSearch.scala (hunk 2 above)
Comment (AndySpaven) [reply app/uk/gov/hmrc/thirdpartyapplication/models/ApplicationSearch.scala:148]: Yep. Sorted

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**